### PR TITLE
WID-667: Feat: Actions Settings page

### PR DESCRIPTION
**Pull request by andy-t-wang**
Screenshot 2024-01-30 at 8 58 13 AM
Screenshot 2024-01-30 at 8 58 15 AM

1. Add Mini Kiosk
2. Add Code Display -> Slight WIP might try to make the code prettier in the future
3. Will add Typography and Size once I merge since I need to change every branch
4. Moved types out of legacy

### CODE REVIEW

File: web-app-router/scenes/Portal/Teams/TeamId/Apps/AppId/Actions/ActionId/Settings/UpdateAction/index.tsx [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+            name="maxVerifications"
+            control={control}
+            render={({ field }) => {
+              return (
+                <MaxVerificationsSelector
+                  value={field.value}
+                  onChange={field.onChange}
+                  errors={errors.maxVerifications}
+                  showCustomInput
+                  className="w-136" // border is 2 px
```
Comment (igorosip0v) [inline web-app-router/scenes/Portal/Teams/TeamId/Apps/AppId/Actions/ActionId/Settings/UpdateAction/index.tsx:142]: @andy-t-wang Why do we need to hardcode input's height? let's make it to just use parent's width `w-full`

File: web-app-router/scenes/Portal/Teams/TeamId/Apps/AppId/Actions/ActionId/Settings/page/index.tsx [hunk 2]
```diff
@@ -1,7 +1,62 @@
...
+            variant="secondary"
+            href="https://docs.worldcoin.org/id/idkit"
+            className="text-grey-700 py-3 px-7 "
+          >
+            <DocsIcon />
+            Learn more
+          </DecoratedButton>
+        </div>
+        <hr className="my-5 w-full text-grey-200 border-dashed" />
+        <div className="w-full grid-cols-2 grid items-start justify-between gap-x-32">
```
Comment (igorosip0v) [inline web-app-router/scenes/Portal/Teams/TeamId/Apps/AppId/Actions/ActionId/Settings/page/index.tsx:55]:
```suggestion
        <div className="w-full grid-cols-1fr/auto grid items-start justify-between gap-x-32">
```

Comment (igorosip0v) [reply web-app-router/scenes/Portal/Teams/TeamId/Apps/AppId/Actions/ActionId/Settings/page/index.tsx:55]: In design, it's not two equal columns. But to make this work you need to add something like min-w-[480px] to `TryAction`

**Review by igorosip0v — APPROVED** [review]

File: web-app-router/scenes/Portal/Teams/TeamId/Apps/AppId/Actions/ActionId/Settings/TryAction/MiniKiosk/Waiting/index.tsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+            className="h-aut max-w-full w-full "
+            value={qrData}
+          />
+        )}
+        {!qrData && <div>Loading QR code...</div>}
+        <Button
+          type="button"
+          onClick={handleCopy}
+          disabled={!qrData}
+          className="h-9 font-medium text-14 flex flex-row items-center"
```
[the remark this answers is not in the document]
Comment (andy-t-wang) [reply web-app-router/scenes/Portal/Teams/TeamId/Apps/AppId/Actions/ActionId/Settings/TryAction/MiniKiosk/Waiting/index.tsx:40]: Noted! Typography will be used when merged in feature branch

File: web-app-router/scenes/Portal/Teams/TeamId/Apps/AppId/Actions/ActionId/Settings/TryAction/MiniKiosk/Waiting/index.tsx [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    navigator.clipboard
+      .writeText(qrData)
+      .then(() => setCopied(true))
+      .then(() => new Promise((resolve) => setTimeout(resolve, 3000)))
+      .finally(() => setCopied(false));
+  }, [qrData]);
+
+  return (
+    <div className="grid gap-y-5">
+      <div className="flex flex-col items-center portrait:py-12 landscape:py-6 bg-white p-10 rounded-2xl gap-y-5 shadow-qrCode">
```
[the remark this answers is not in the document]
Comment (andy-t-wang) [reply web-app-router/scenes/Portal/Teams/TeamId/Apps/AppId/Actions/ActionId/Settings/TryAction/MiniKiosk/Waiting/index.tsx:27]: the spacing does seem to be 5 in the designs

**Pull request merged**